module/icp/asm-x86_64/modes/aesni-gcm-x86_64.S

```diff
@@ -363,7 +366,8 @@ _aesni_ctr32_ghash_6x:
 	vpxor	16+8(%rsp),%xmm8,%xmm8
 	vpxor	%xmm4,%xmm8,%xmm8
 
-	.byte	0xf3,0xc3
+	# .byte	0xf3,0xc3
```

Comment:
I'd prefer to see all commented out .byte instruction removed, they have no purpose and could be misleading.